VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/vho-hearing-list/vho-hearing-list.component.ts

```diff
@@ -45,4 +45,14 @@ export class VhoHearingListComponent implements OnInit {
         const hearing = new ConferenceResponseVho({ id: conference.id, scheduled_date_time: conference.scheduledDateTime, status: conference.status });
         return new Hearing(hearing);
     }
+
+    mapToHearingWithParticipants(conference: HearingSummary): Hearing {
```

Comment:
It doesn't seem necessesary to duplicate `mapToHearing` with the addition of adding participants.
Either merge the two methods or call `mapToHearing` in `mapToHearingWithParticipants` to avoid code duplication

Reply:
done